internal/helm/helm.go

```diff
@@ -141,9 +146,37 @@ epinio:
 
 	logger.Info("app helm setup", "parameters", yamlParameters)
 
+	client, err := GetHelmClient(parameters.Cluster.RestConfig, logger, parameters.Namespace)
+	if err != nil {
+		return err
+	}
+
+	helmChart := appChart.HelmChart
+	helmVersion := ""
+
+	// See also part.go, fetchAppChart
```

Comment:
There is duplication with this code: https://github.com/epinio/epinio/blob/1270-multi-app-chart-support/internal/api/v1/application/part.go#L158

That method needs to be split in functions with smaller scope. Maybe then we can reuse code here.

Reply:
Let's not keep 3 PRs open for this. We can try that on a separate PR.